explainability-connectors/src/main/java/org/kie/trustyai/connectors/grpc/GrpcLoadBalancing.java

```diff
@@ -0,0 +1,23 @@
+package org.kie.trustyai.connectors.grpc;
+
+/**
+ * Enum for gRPC load balancing policies
+ */
+public enum GrpcLoadBalancing {
+    ROUND_ROBIN("round_robin");
```

Comment:
I would add also the `pick_first` strategy so that we already have the value here
```suggestion
    ROUND_ROBIN("round_robin"),
    PICK_FIRST("pick_first");
```